src/validators/union.rs

```diff
@@ -265,7 +298,7 @@ impl BuildValidator for TaggedUnionValidator {
         };
 
         let key = intern!(py, "from_attributes");
-        let from_attributes = schema_or_config(schema, config, key, key)?.unwrap_or(false);
+        let from_attributes = schema_or_config(schema, config, key, key)?.unwrap_or(true);
```

Comment:
this is causing a test to fail.

Please can you also update `core_schema.py` to reflect this change, a comment is enough.

Reply:
I'll just revert this change and defer to https://github.com/pydantic/pydantic-core/pull/398 for a better treatment

Reply:
makes sense.

Reply:
(I undid the undo but addressed Samuel's original comment)